Cargo.toml

```
@@ -14,8 +14,7 @@ lib-profile-release = "wasm-release"
 
 [dependencies]
 itertools = "0.13.0"
-#simplicityhl = "0.2.0"
-simplicityhl = { package = "simfony", git = "https://github.com/BlockstreamResearch/simplicityhl", rev = "d5284014e9f67593e50b272f1f676ea8d09f6ec8" }
+simplicityhl = { version = "0.3.0", path = "/home/apoelstra/code/BlockstreamResearch/SimplicityHL/master" }
```

This isn't compiling :) 

```suggestion
simplicityhl = { version = "0.3.0" }
```

---

lol oops